app/components/Application/ApplicationDetailsCardItemCustomFields.tsx

```diff
@@ -29,6 +29,9 @@ const customFields = (
   };
 
   const CUSTOM_FIELDS: Record<string, React.FunctionComponent<FieldProps>> = {
+    ProblemReportField: (props) => (
+      <span id={props.idSchema?.$id}>{props.formData ?? null}</span>
+    ),
```

Comment:
No sure that you need this now that you're handling the conditional rendering in the `FormFieldTemplate`

Reply:
the `?? null` part is redundant, but the `FormFieldTemplate` passes the children to be rendered as-is from what I understand, so we still need a case for that type of field (or it'll get caught by the `StringField` case which is what we wanted to depart from)

Reply:
The departure from `StringField` was because that component renders `[no data entered]` when the value is null-ish. Now that there is the uiSchema option, I think that we can use `StringField` again.

Reply:
True - and then we get the diff logic already showing up :)